src/Parser/Mage/Arcane/CHANGELOG.js

```diff
@@ -1,54 +1,63 @@
+import React from 'react';
+
 import { Sharrq } from 'CONTRIBUTORS';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
 
 export default [
+  {
+    date: new Date('2018-08-28'),
+    changes: <React.Fragment>Added support for <SpellLink id={SPELLS.GALVANIZING_SPARK.id} /> and <SpellLink id={SPELLS.ANOMALOUS_IMPACT.id} /></React.Fragment>,
+    contributors: [Sharrq],
+  },
   {
     date: new Date('2018-08-11'),
-    changes: 'Added Arcane Missiles Module and Time Anomaly Mana Management.',
+    changes: <React.Fragment>Added Arcane Missiles Module and Time Anomaly Mana Management.</React.Fragment>,
```

Comment:
You only need to use <React.Fragment> if there are tooltips in the text, otherwise string is fine :)
You could add tooltips to a lot of the changelog entries though if you wanted.